ctlevent.js

```diff
@@ -131,8 +141,14 @@ CTLEvent.prototype.render = function() {
         returnString += 'Room ' + this.roomNumber + ', ';
     } 
 
-    returnString += this.location + '</br>' +  
-        '</div><div class="event_properties">' +
+    returnString += this.location + '</br></div>';
```

Comment:
The `</br>` here can be removed - i'm not sure what tag this is closing, and this is unnecessary. `<br>` doesn't need an end tag in HTML. Also, the `</br>` at line 135 can be removed as well.